events_test/src/main/java/com/adobe/aio/event/management/RegistrationServiceTester.java

```diff
@@ -99,7 +103,6 @@ private static void assertUrl(String stringUrl) {
     try {
       Assert.assertNotNull(stringUrl);
       URL url = new URL(stringUrl);
-      Assert.assertEquals("https", url.getProtocol());
```

Comment:
Why is this removed?

Reply:
to have the test running against a local non https url

Reply:
ok, got it